pandas/core/arrays/categorical.py

```diff
@@ -1846,6 +1846,11 @@ def __iter__(self):
         """Returns an Iterator over the values of this Categorical."""
         return iter(self.get_values().tolist())
 
+    def __contains__(self, key):
+        """Returns True if `key` is in this Categorical."""
+        code_values = self.categories.get_indexer([key])
```

Comment:
``get_loc`` is much faster than ``get_indexer`` as it returns a slice object versus a numpy array.